clarity/contracts/multisig/multisig-crp-wbtc-59760-usda.clar

```diff
@@ -0,0 +1,280 @@
+(impl-trait .trait-multisig-vote.multisig-vote-trait)
```

Comment:
This one can be fixed as I changed on #182 pull Request, if you are okay, let me push to this branch